yarn-project/acir-simulator/src/avm/opcodes/external_calls.ts

```diff
@@ -55,23 +87,47 @@ export class Call extends Instruction {
 
 export class StaticCall extends Instruction {
   static type: string = 'STATICCALL';
-  static numberOfOperands = 7;
+  static readonly opcode: Opcode = Opcode.STATICCALL;
+
+  // Instruction wire format with opcode.
```

Comment:
I see this comment for each wireFormat. Could you make this comment just a bit clearer about the purpose of wireFormat? Something like "informs (de)serialization regarding how to interpret each byte/operand."